### PR TITLE
docs: Default to nodejs14.x in AWS docs

### PR DESCRIPTION
**Pull request by tobiastornros**
Changes deafult nodejs version from 12.x to 14.x in docs for AWS provider.
AWS is ending support fon nodejs12.x during late 2022. Read more here: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtime-support-policy

### CODE REVIEW

File: docs/providers/aws/guide/functions.md [hunk 1]
```diff
@@ -684,7 +685,8 @@ service: myService
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
+
```
Comment (mnapoli) [inline docs/providers/aws/guide/functions.md:689]: Extra line

File: docs/providers/aws/guide/functions.md [hunk 2]
```diff
@@ -132,7 +132,8 @@ service: myService
...
```
Comment (mnapoli) [inline docs/providers/aws/guide/functions.md:136]: Extra line to remove

**Review by mnapoli — COMMENTED** [review]
Thank you, that sounds like a good idea indeed. I added a couple inline comments about unrelated changes 👍

**Comment by tobiastornros** [comment]
> Thank you, that sounds like a good idea indeed. I added a couple inline comments about unrelated changes 👍

Should be fixed now :)

**Review by mnapoli — APPROVED** [review]
Looking good, thank you!

**Pull request merged**